pkg/contrib/src/components/Interactive.ts

```diff
@@ -2,22 +2,22 @@ import { Component } from "@piggo-legends/core";
 
 export type Bounds = { x: number, y: number, w: number, h: number };
 
-export type InteractiveProps = {
+export type ClickableProps = {
   width: number;
   height: number;
   active: boolean;
   onPress: string; // action ID
 }
 
-export class Interactive implements Component<"interactive"> {
-  type: "interactive";
+export class Clickable implements Component<"clickable"> {
+  type: "clickable";
```

Comment:
the fundamental notion of "clickable" needs reconsidered

how to handle user flows Hover -> Click / Button

Reply:
#42